Objects/abstract.c

```diff
@@ -199,6 +199,30 @@ PyObject_GetItem(PyObject *o, PyObject *key)
     return type_error("'%.200s' object is not subscriptable", o);
 }
 
+int
+PyMapping_GetOptionalItem(PyObject *o, PyObject *key, PyObject **pvalue)
```

Comment:
I suggest to rename "o" to "obj", as in the doc. I dislike single letter variable names.